tools/clang/test/CodeGenHLSL/unroll/count3.hlsl

```diff
@@ -0,0 +1,29 @@
+// RUN: %dxc -E main -T ps_6_0 %s | FileCheck %s
```

Comment:
I suggest renaming `count_greater_than_loop_iterations.hlsl` or something that makes it clear what the test is meant to exercise.